scripted/sbt/src/main/scala/sbt/test/SbtHandler.scala

```diff
@@ -80,7 +80,10 @@ final class SbtHandler(directory: File,
     val thread = new Thread() { override def run() = { p.exitValue(); server.close() } }
```

Comment:
You get 5 workers for free but the turnaround time is at minimum 45 minutes for every commit you do, and there's usually a long queue. We need more powerful alternatives. I will not insist on using our free CI, but can you please consider paying a pro account in Travis to increase the CI resources?

Being honest here, contributing to sbt is less than an ideal experience. I've tried solving it by parallelizing the testing engine, but even if that is blocked by Travis resources, I don't know what else to do. In my view, there is no other way to move forward than increasing the resources.